libraries/src/MVC/Model/AdminModel.php

```diff
@@ -721,7 +722,15 @@ public function checkin($pks = array())
 			}
 			else
 			{
-				$this->setError($table->getError());
+				if ($table instanceof Table)
```

Comment:
In this case we're fine I think. We're probably just going to throw an exception rather than return false.

Reply:
This will probably break something. I am not sure how to deal with exceptions when doing the semi conversion (keeping the Models, converting the Tables). Maybe we should have the `getError` `setError` functionality in the EntityTableTrait for this stage of conversion.